src/App.js

```diff
@@ -1,60 +1,77 @@
-import React from 'react';
+import React, {lazy, Suspense} from 'react';
 import {
   BrowserRouter as Router,
   Switch,
   Route,
   Redirect,
 } from 'react-router-dom';
 import * as Icon from 'react-feather';
-
+import Spinner from './components/spinner';
 import './App.scss';
-import Home from './components/home';
-import Navbar from './components/navbar';
-import Links from './components/links';
-import Summary from './components/summary';
-import Cluster from './components/cluster';
-import FAQ from './components/faq';
-import Banner from './components/banner';
+const Home = lazy(() => import('./components/home'));
+const Navbar = lazy(() => import('./components/navbar'));
```

Comment:
Navbar is needed for all pages. So why should it be split into chunk?

Reply:
we may not need to be lazy.I just multi selected and wrote.

Reply:
made it non-lazy